common/data_refinery_common/rna_seq.py

```diff
@@ -28,13 +28,16 @@
 EARLY_TXIMPORT_MIN_PERCENT = 0.80
 
 
-def get_latest_organism_index(organism):
+def get_latest_organism_indices(organism):
     # Salmon version gets saved as what salmon outputs, which includes this prefix.
     current_salmon_version = "salmon " + get_env_variable("SALMON_VERSION", "0.13.1")
+    indices = OrganismIndex.objects.filter(
+        salmon_version=current_salmon_version, organism=organism
+    ).order_by("-created_at")
```

Comment:
Not sure whether the `order_by()` needs to be (re-)moved.

Reply:
I just tested that the order_by is preserved on staging. We don't need to move it to after the filter.